pkg/reporter/reporter_test.go

```diff
@@ -255,6 +255,16 @@ func TestJSONReporterPublish(t *testing.T) {
 	expect := `{
   "violations": [
     {
+      "location": {
```

Comment:
If we don't like the JSON output changed, we can easily create a custom serializer with jsoniter